src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
```

Comment:
It's not a `pressedKey`, it's the message you show on the page. I suppose it's better to store in the state `pressedKey ` value (default = ' ' or null) and render the page according to its state.

Reply:
fix @Anastasiia-Khudych comment